src/components/CTable.vue

```diff
@@ -3,7 +3,10 @@
     <table class="table-fixed w-full">
       <thead>
         <tr class="bg-gray-100 border-4">
-          <th class="border-4 py-2">Message Id</th>
+          <th class="border-4 py-2">
```

Comment:
This makes us think we can sort the message are sorted. It would be cool to be able to sort a column by clicking on the header. Maybe remove this and create an issue about that ?